stubs/seaborn/seaborn/regression.pyi

```diff
@@ -25,15 +28,15 @@ def lmplot(
     height: float = 5,
     aspect: float = 1,
     markers: str = "o",
-    sharex: bool | Literal["col", "row"] | None = None,
-    sharey: bool | Literal["col", "row"] | None = None,
+    sharex: bool | Literal["col", "row"] | None = None,  # deprecated
+    sharey: bool | Literal["col", "row"] | None = None,  # deprecated
     hue_order: Iterable[str] | None = None,
     col_order: Iterable[str] | None = None,
     row_order: Iterable[str] | None = None,
     legend: bool = True,
-    legend_out: Incomplete | None = None,
-    x_estimator: Incomplete | None = None,
-    x_bins: Incomplete | None = None,
+    legend_out: bool | None = None,  # deprecated
+    x_estimator: Callable[[Incomplete], Incomplete] | None = None,
```

Comment:
I am not sure how to type the argument and return type of this callable. It is [documented](https://seaborn.pydata.org/generated/seaborn.regplot.html) as "callable that maps vector -> scalar", something like `numpy.mean` but I think it will require Higher-Kinded TypeVars and a lot of overloads to get it right (unless I am missing something trivial about it)